### PR TITLE
[api-minor] Add a parameter to `PDFPageProxy_getTextContent` that controls whether `PartialEvaluator_getTextContent` will attempt to combine same line text items

### PR DESCRIPTION
**Pull request by Snuffleupagus**
From the discussion in issue #7445, it seems that there may be cases where an API consumer would want to get the text content as is, without combined text items.

### CODE REVIEW

File: src/core/evaluator.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* Copyright 2012 Mozilla Foundation
+  /* Copyright 2012 Mozilla Foundation
```
Comment (timvandermeij) [inline src/core/evaluator.js]: Nit: this change does not seem necessary

Comment (Snuffleupagus) [inline src/core/evaluator.js]: Thanks, it certainly shouldn't be there, good catch! I've got no idea how that happened :-)

**Comment by yurydelendik** [comment]
r+ with making combineTextItems true by default at the API level (perhaps making it disableCombineTextItems and use combineTextItems at the core)

**Comment by Snuffleupagus** [comment]
@yurydelendik Thanks for the review! Is the updated patch more what you had in mind?

/botio test

**Comment by yurydelendik** [comment]
Thank you for the patch.

**Pull request merged**